### ISSUE TITLE
Search - Simple Search

### ISSUE THREAD

**Issue by fashxp** [pimcore/studio-ui-bundle#245]
Global Simple Search with combined search result. 

![image](https://github.com/pimcore/studio-ui-bundle/assets/8792145/87b5663e-89f7-4c3f-8133-aa24cabe5b0b)

https://www.figma.com/design/kTxowT29eCHwNh5NspDGGu/Pimcore-Components?node-id=2972-110571&t=TH6LzmPptxSxTMON-4

![image](https://github.com/pimcore/studio-ui-bundle/assets/8792145/0f22b48a-1d9e-4b1c-b08c-16e2351806d9)

https://www.figma.com/design/kTxowT29eCHwNh5NspDGGu/Pimcore-Components?node-id=2972-110573&t=TH6LzmPptxSxTMON-4

### For element specific search results see 
- https://github.com/pimcore/studio-ui-bundle/issues/246
- https://github.com/pimcore/studio-ui-bundle/issues/247

### See also
- https://github.com/pimcore/product-management/issues/523

**Comment by lukmzig** [comment]
Currently on hold, waiting for data objects grid endpoint.
 The logic for filtering the correct columns needs to be finished first so we can create an endpoint (similiar to `/pimcore-studio/api/data-object/grid/available-columns/{classId}/{folderId}`) to be able to filter columns based on the `Visible in Search Result`

![Image](https://github.com/user-attachments/assets/454c5194-792f-41ae-9531-b42597830e92)

See task: https://github.com/pimcore/studio-backend-bundle/issues/691
